survey-sdk/src/main/java/co/nimblehq/survey/sdk/constant/Constant.kt

```diff
@@ -0,0 +1,5 @@
+package co.nimblehq.survey.sdk.constant
+
+object Constant {
+    const val AUTHORIZATION_HEADER = "Authorization"
+}
```

Comment:
@doannimble "Constants should be declared at the top-level scope and placed at the bottom of the file." 😄  Kotlin convention on [Compass](https://nimblehq.co/compass/development/code-conventions/kotlin/)
![image](https://user-images.githubusercontent.com/6950766/134617062-6c19ef20-fe3a-494d-bb96-769769408d51.png)

Reply:
@hoangnguyen92dn That is for `Class`. I'm using `Object`. 
<img width="778" alt="Screen Shot 2021-09-24 at 11 17 32 AM" src="https://user-images.githubusercontent.com/88996251/134617615-857b9915-e55f-4f50-b50e-42a2e878b961.png">

I'm not sure if we only accept the way to create the `Constant` as stand alone 🤔

Reply:
@doannimble The same. We could define `const val` directly instead of wrapping by `Object` 😁

Reply:
@hoangnguyen92dn Updated